apps/full-site-editing/full-site-editing-plugin/full-site-editing/utils/class-a8c-wp-template-data-inserter.php

```diff
@@ -0,0 +1,105 @@
+<?php
+/**
+ * Template data inserter file.
+ *
+ * @package full-site-editing
+ */
+
+/**
+ * Class A8C_WP_Template_Data_Inserter
+ */
+class A8C_WP_Template_Data_Inserter {
+	/**
+	 * This function will be called on plugin activation hook.
+	 */
+	public function insert_default_template_data() {
+		$header_id = wp_insert_post(
+			[
+				'post_title'     => 'Header',
+				'post_content'   => $this->get_header_content(),
+				'post_status'    => 'publish',
+				'post_type'      => 'wp_template_part',
+				'comment_status' => 'closed',
+				'ping_status'    => 'closed',
+			]
+		);
+
+		if ( ! term_exists( 'header', 'wp_template_part_type' ) ) {
+			wp_insert_term( 'header', 'wp_template_part_type' );
+		}
+
+		wp_set_object_terms( $header_id, 'header', 'wp_template_part_type' );
+
+		$footer_id = wp_insert_post(
+			[
```

Comment:
Have we arrived at using `[]` syntax rather than `array()`? If so, we should probably be consistent.

Reply:
Indeed, the plugin's minimum PHP version is [5.6+](https://github.com/Automattic/wp-calypso/blob/master/apps/full-site-editing/full-site-editing-plugin/readme.txt#L7), so we're in the clear 👍

Reply:
Yes, I think it's being used throughout the plugin now. If there are any remaining references to `array()` they should be updated too.